pkgs/development/python-modules/fast-histogram/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, buildPythonPackage
+, pytestCheckHook
+, fetchFromGitHub
+, pythonOlder
```

```suggestion
, pythonOlder
, python
```

Sorry, `nixpkgs-hammer` erroneously flagged this as unused

---

Ok, I added.